Sources/BraintreeUIComponents/VenmoButton.swift

```diff
@@ -1,47 +1,74 @@
 import SwiftUI
+import BraintreeVenmo
```

Comment:
iirc you will need to do this to make cocoapods happy:
```suggestion
#if canImport(BraintreeCore)
import BraintreeCore
#endif
```

Reply:
Oh awesome thanks! Was trying to figure out how to resolve this